graph/common/Storage.java

```diff
@@ -56,6 +57,26 @@ default Schema asSchema() {
         throw exception(TypeDBException.of(ILLEGAL_CAST, className(this.getClass()), className(Schema.class)));
     }
 
+    class SortedPair<T extends Bytes.ByteComparable<T>, U> implements Bytes.ByteComparable<SortedPair<T, U>> {
```

Comment:
this lets us iterate over a pair that comes from storage while meeting the new requirement that all constructors that provided to `RockIterator`s are `ByteComparable`